src/components/Listing.tsx

```diff
@@ -1,29 +1,260 @@
 import React from "react";
 import { ListingItem } from "./ListingItem";
-import { useSelector } from "react-redux";
+import { useSelector, useDispatch } from "react-redux";
 import { ApplicationState } from "../app/store";
-export const Listing: () => JSX.Element = () => {
-  const items = useSelector((state: ApplicationState) => state.items);
-  const tabs = [
-    {
-      title: "Links to read",
-      items: [...items.filter((item) => !item.alreadyRead)],
-      visible: true,
+import { Tab as TabPanel, Tabs } from "@material-ui/core";
+import { makeStyles, Theme } from "@material-ui/core/styles";
+import { selectTab } from "../features/tabs";
+import { ListItem } from "../types/ListItem";
+import { COLORS } from "../utils/constants";
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
This is monstrosity and I am gonna work on it when styling the app.

Reply:
Why do we then have it now here when we are not doing styling properly now?